packages/account/src/App.tsx

```diff
@@ -19,8 +20,10 @@ const App = ({ passthrough }: TAppProps) => {
 
     return (
         <StoreProvider store={root_store}>
-            <Routes />
-            <ResetTradingPassword />
+            <APIProvider>
```

Comment:
@likhith-deriv `@deriv/stores` uses `@deriv/api`, `APIProvider` should wrap `StoreProvider`.